src/templates/fr/2020/methodology.html

```diff
@@ -360,39 +365,85 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
+      </p>
+    </section>
+
+    <section>
+        <h4 id="parsel"><a href="#parsel" class="anchor-link">Parsel</a></h4>
+
+        <p>
+            <a href="https://projects.verou.me/parsel/">Parsel</a> est un analyseur de sélecteur CSS et un calculateur de spécificité, écrit à l’origine par la responsable du chapitre <a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a>, <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a>, et dont elle a ouvert le code source dans une bibliothèque distincte.
+            Il est largement utilisé pour tous les indicateurs du CSS qui concernent les sélecteurs et la spécificité.
+        </p>
+    </section>
+
     <section>
       <h2 id="process"><a href="#process" class="anchor-link">Processus analytique</a></h2>
 
       <p>
-        La planification et l’exécution du Web Almanac ont pris environ un an,  avec la coordination de dizaines de contributeurs de la communauté Web.  Cette section décrit pourquoi nous avons choisi les indicateurs que vous voyez dans le Web Almanac, comment ils ont été consultés et interprétés.
+        La planification et l’exécution du Web Almanac ont pris environ un an, avec la coordination de plus d’une centaine de <a href="{{ url_for('contributors', year=year, lang=lang) }}">contributeurs et contributrices</a> de la communauté Web. Cette section décrit pourquoi nous avons choisi les chapitres que vous voyez dans le Web Almanac, comment leurs mesures ont été interrogées et comment elles ont été interprétées.
       </p>
     </section>
 
     <section>
-      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">Réflexion</a></h3>
+      <h3 id="planning"><a href="#planning" class="anchor-link">Planification</a></h3>
+
+      <p>
+        Le Web Almanach 2020 a été lancé en <a href="https://twitter.com/rick_viscomi/status/1273135952848977920">juin 2020</a>, un peu plus tard qu’en 2019 en raison des troubles liés au COVID-19 et des manifestations en faveur de la justice sociale. Ces événements et d’autres de 2020 ont été sous-jacents tout au long du processus de production et ont mis beaucoup de pression supplémentaire sur nos contributeurs et contributrices, au-delà du stress d’un projet aussi effréné que celui-ci.
+      </p>
+
+      <p>
+        Comme nous l’avons <a href="https://almanac.httparchive.org/en/2019/methodology#brainstorming">déclaré dans la Méthodologie de l’année dernière</a>:
+      </p>
+
+      <blockquote>
+        L’un des objectifs explicites des prochaines éditions du Web Almanac est d’encourager encore plus l’inclusion de voix sous-représentées et hétérogènes comme auteurs, autrices, relecteurs et relectrices.
+      </blockquote>
 
       <p>
```

Comment:
Not sure if a `ul` could be nested in a `p`. Maybe close the `p` before, then `ul`, then another `p` ?

Reply:
I think you're right to doubt but I merely reproduce here the EN code layout, so I'm not going to fix this here.

Reply:
Yes, this is not an important point, more an old reflex of an old-XHTML grummy :D

Reply:
Good shout! Our Linter doesn't complain but the [Nu Validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Falmanac.httparchive.org%2Fen%2F2020%2Fmethodology) does! Will fix in a follow up PR after merging this.